org.lflang/src/org/lflang/generator/GeneratorBase.xtend

```diff
@@ -106,7 +106,7 @@ abstract class GeneratorBase extends AbstractLFValidator implements TargetTypes
     /**
      * Constant that specifies how to name generated delay reactors.
      */
-    public static val GEN_DELAY_CLASS_NAME = "__GenDelay"
+    public static val GEN_DELAY_CLASS_NAME = "_GenDelay"
```

Comment:
Maybe this should be `_lf_GenDelay` to be consistent with other LF-related reserved keywords in C.

Reply:
Good idea. Done.